CMakeLists.txt

```diff
@@ -42,6 +42,7 @@ option(USE_NCCL "Use NCCL" ON)
 option(USE_NERVANA_GPU "Use Nervana GPU backend" OFF)
 option(USE_NNPACK "Use NNPACK" ON)
 option(USE_OBSERVERS "Use Observer Library" OFF)
+option(USE_ONNX "Use ONNX" ON)
```

Comment:
we might want to get rid of this later on if op schemas depend on onnx. In that case it won't be an optional dependency any more